### DISCUSSION TITLE
Webpack6 module federation and backwards compatibility

### DISCUSSION THREAD

**Discussion by beetz12** [webpack/webpack#16922] [Q&A] [answered]
Currently we're using module federation with webpack to run multiple versions of Angular micro-frontends in our Angular app shell. When webpack6 comes out, we're wondering if it'll still be possible to support two different versions of Angular and if it'll be backward compatible with webpack5.

**Comment by alexander-akait** [comment, marked as answer]
I would say - yes, we do not have plans to drastically rewrite the logic, only new features and bug fixes

**Reply by beetz12** [reply]
Awesome, thank you @alexander-akait !
